tiledb/api/src/query/mod.rs

```diff
@@ -44,10 +44,10 @@ impl Drop for RawQuery {
     }
 }
 
-pub trait Query {
-    fn base(&self) -> &QueryBase;
+pub trait Query<'a> {
```

Comment:
Can we call 'a 'array instead? Is it meant to represent the lifetime of the array?

Reply:
It is, yep. Happy to do that.  Most of the standard library uses `'a` but I agree that the lifetime names ought to be more verbose.

Reply:
I'm fine with a comment which makes it clear instead of renaming everything